lib/driverProviders/sauce.ts

```diff
@@ -63,10 +63,11 @@ export class Sauce extends DriverProvider {
     this.config_.capabilities['username'] = this.config_.sauceUser;
     this.config_.capabilities['accessKey'] = this.config_.sauceKey;
     this.config_.capabilities['build'] = this.config_.sauceBuild;
-    let auth = 'https://' + this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
-    this.config_.seleniumAddress = auth +
-        (this.config_.sauceSeleniumAddress ? this.config_.sauceSeleniumAddress :
-                                             'ondemand.saucelabs.com:443/wd/hub');
+
+    let auth = this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
+    this.config_.seleniumAddress = this.config_.sauceSeleniumAddress ?
+        'http://' + auth + this.config_.sauceSeleniumAddress :
```

Comment:
I suppose where I was going with that is that is that we need to support `http`, but in this solution, we're now guaranteeing that https is just as impossible as http is, now, if there is a path that has been unconsidered, thus far.

Would a config flag work for you?